packages/pyodide-kernel/src/kernel.ts

```diff
@@ -62,7 +62,7 @@ export class PyodideKernel extends BaseKernel implements IKernel {
       baseUrl,
       pyodideUrl,
       indexUrl,
-      pipliteWheelUrl: pipliteWheelUrl.default,
+      pipliteWheelUrl: options?.pipliteWheelUrl || pipliteWheelUrl.default,
```

Comment:
Wondering if the `?` is needed here, since `options` is not optional?

```suggestion
      pipliteWheelUrl: options.pipliteWheelUrl || pipliteWheelUrl.default,
```